ingest-app/docs/api.md

```diff
@@ -54,6 +54,11 @@ See the [CMR Client Partner User Guide](https://wiki.earthdata.nasa.gov/display/
         * [DELETE - Delete a subscription.](#delete-subscription)
         * [Subscription Access Control](#subscription-access-control)
 * %GENERIC-TABLE-OF-CONTENTS%
+* [Publish Drafts](#publish-drafts)
+    * [/publish/\<non-variable-draft-concept-id\>/\<native-id\>](#publish-non-variable-draft-endpoint)
+        * [PUT - Publish a non variable draft.](#publish-non-variable-draft)
```

Comment:
Change the text to state 
- Publish all draft records except for variables.

Reply:
done.